common-docs/teachertool/validator-plans-shared.json

```diff
@@ -150,6 +150,19 @@
                     }
                 }
             ]
+        },
+        {
+            ".desc": "a number exists // no catalog entry",
```

Comment:
nit: I don't see a need to specify "no catalog entry" here. I think that'll end up being true for a lot of validator plans.